web/app/widget-wrappers/sticker-picker/sticker-picker.component.html

```diff
@@ -27,7 +27,7 @@
             <div class="stickers">
                 <div class="sticker" *ngFor="let sticker of pack.stickers trackById"
                      (click)="sendSticker(sticker, pack)">
-                    <img [src]="getThumbnailUrl(sticker.thumbnail.mxc, 48, 48)" width="48" height="48" class="image"
+                    <img [src]="getThumbnailUrl(sticker.thumbnail.mxc, 48, 48)" class="image"
```

Comment:
We can't lose the width and height because the media repo is not guaranteed to return the size we requested. It can return (much) larger images.

Reply:
It seems like that the thumbnails never load as 48x48 but as 240x240 on my end anyway. The CSS guarantees that the image always resizes itself to fit in the sticker square for all screen sizes as setting 48x48 statically made it very small on mobile. I can re-add height/width HTML back, but from trying it out quickly it doesn't have any effect as the CSS already overrides it.

Reply:
hmm, I'll take a look at this too then.